src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -959,12 +959,14 @@ await using (var writer = new AsyncBlittableJsonTextWriter(docsContext, _buffer)
                                 writer.WritePropertyName(docsContext.GetLazyStringForFieldWithCaching(IncludedCounterNamesSegment));
                                 writer.WriteIncludedCounterNames(includeCountersCommand.CountersToGetByDocId);
 
-                                var size = includeCountersCommand.CountersToGetByDocId.Sum(kvp => kvp.Key.Length + kvp.Value.Sum(name => name.Length)) //CountersToGetByDocId
-                                    + includeCountersCommand.Results.Sum(kvp => kvp.Value.Sum(counter => counter.CounterName.Length
-                                                                                                         + counter.DocumentId.Length
-                                                                                                         + sizeof(long) //Etag
-                                                                                                         + sizeof(long) //Total Value
-                                    ));
+                                var size = includeCountersCommand.CountersToGetByDocId.Sum(kvp =>
+                                               kvp.Key.Length + kvp.Value.Sum(name => name.Length)) //CountersToGetByDocId
+                                    + includeCountersCommand.Results.Sum(kvp =>
+                                        kvp.Value.Sum(counter => counter == null ? 0 : counter.CounterName.Length
```

Comment:
When `counter` can be `null` here?

Reply:
When the counter does not exist.

Reply:
Oh, I see. Indeed the test name says "EvenIfTheyDoNotExist" :) Thanks for the fix